static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -642,3 +644,30 @@ $hosted-stripe: 28px;
 .hosted__popup-text {
     font-weight: 200;
 }
+
+.hosted__central-play-button {
+    @include f-headlineSans;
+    position: absolute;
+    top: calc(50% - 18px);
+    left: calc(50% - 71px);
+    border: 2px solid #ffffff;
+    height: 40px;
+    width: 146px;
+    border-radius: 30px;
+    color: #ffffff;
+    line-height: 42px;
```

Comment:
Include different headline mixin if you want to change font-size and line-height values.